test/IntegrationTests/MSTest.Acceptance.IntegrationTests/SdkTests.cs

```diff
@@ -14,18 +14,57 @@ public sealed class SdkTests : AcceptanceTestBase
 {
     private const string AssetName = "MSTestSdk";
 
-    public SdkTests(ITestExecutionContext testExecutionContext, AcceptanceFixture acceptanceFixture)
+    private const string ClassicSourceCode = """
```

Comment:
I am not sure what Classic should mean here. I find it more confusing that e.g. SingleTestSourceCode